frontend/src/app/aethel/aethel.component.html

```diff
@@ -0,0 +1,121 @@
+<h2 class="title is-2" i18n>Æthel</h2>
+
+<p class="field" i18n>
+    Welcome to Æthel. Search for a word or lemma in the input field below to
+    start.
+</p>
+
+<form class="form">
```

Comment:
(extremely nitpicky) since you already have a form, you could use the submit event instead of defining key down and button click separately

Reply:
I tried this before, but I kept running into the issue that the page would be refreshed every time I submitted it.

I just tried again (because I do like tapping into the form submit event and I wanted to figure out how it works), and apparently it only works if you bind a `formGroup` to `<form>`.